src/functionalTest/java/uk/gov/hmcts/probate/functional/businessvalidation/SolCcdServiceBusinessValidationTests.java

```diff
@@ -500,6 +500,19 @@ public void shouldTransformCaseWithCitizenAttributes() {
         assertEquals("1", registrySequenceNumber);
     }
 
+    @Test
+    public void shouldTransformCaseWithTrustCorpAttributes(){
+        String response = transformCase("success.TrustCorpAttribtesSaved.json", TRANSFORM_URL);
+
```

Comment:
COULD: typo success.TrustCorpAttribtesSaved.json, tho it does match the filename

Reply:
Good spot, made the change now